HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentComponentsController.swift

```diff
@@ -815,6 +815,24 @@ extension PaymentComponentsController: PaymentReviewProtocol {
                 return ($0.index ?? 0) < ($1.index ?? 0)
             }
     }
+    
+    /**
+        Retrieves a payment request using the provided payment request ID.
+
+        - Parameter id:         The ID of the payment request to retrieve.
+        - Parameter completion: A closure to be executed once the retrieval is completed, containing the result of the operation as a `PaymentRequest` object on success or a `GiniError` on failure.
+    */
+    public func getPaymentRequest(by id: String, completion: @escaping (Result<PaymentRequest, GiniHealthAPILibrary.GiniError>) -> Void) {
+        giniSDK.getPaymentRequest(by: id) { result in
```

Comment:
We have a public method in  `giniSDK.getPaymentRequest(by: id)`  in `GiniHealth` we don't need to copy it over. We can remove it.